docs/understanding/evaluating/evaluating.md

```diff
@@ -2,12 +2,63 @@
 
 Evaluation and benchmarking are crucial concepts in LLM development. To improve the performance of an LLM app (RAG, agents), you must have a way to measure it.
 
-LlamaIndex offers key modules to measure the quality of generated results. We also offer key modules to measure retrieval quality.
+LlamaIndex offers key modules to measure the quality of generated results. We also offer key modules to measure retrieval quality. You can learn more about how evaluation works in LlamaIndex in our [module guides](/module_guides/evaluating/root.md).
 
-- **Response Evaluation**: Does the response match the retrieved context? Does it also match the query? Does it match the reference answer or guidelines?
-- **Retrieval Evaluation**: Are the retrieved sources relevant to the query?
+## Response Evaluation
 
-You can learn more about how evaluation works in LlamaIndex in our [module guides](/module_guides/evaluating/root.md).
+Does the response match the retrieved context? Does it also match the query? Does it match the reference answer or guidelines? Here's a simple example that evaluates a single response for Faithfulness, i.e. whether the response contains hallucinations:
```

Comment:
I think the i.e. on faithfulness is better if we say "response stays aligned to retrieved context". It can be used to potentially detect hallucinations but not all unfaithful responses are hallucinations I think — in the event that a context might not be accurate and the llm responds with something better, I'm not sure if we call that a hallucination?